test/controllers/DeleteAdditionalMetadataControllerSpec.scala

```diff
@@ -92,13 +111,27 @@ class DeleteAdditionalMetadataControllerSpec extends FrontEndTestHelper {
       val graphQLConfiguration = new GraphQLConfiguration(app.configuration)
       val consignmentService = new ConsignmentService(graphQLConfiguration)
       val customMetadataService = new CustomMetadataService(graphQLConfiguration)
-      val controller =
-        new DeleteAdditionalMetadataController(consignmentService, customMetadataService, getValidJudgmentUserKeycloakConfiguration, getAuthorisedSecurityComponents)
-      val response = controller
-        .confirmDeleteAdditionalMetadata(consignmentId, metadataType(0), fileIds)
-        .apply(FakeRequest(GET, s"/consignment/$consignmentId/additional-metadata/confirm-delete-metadata/${metadataType(0)}"))
+      val displayPropertiesService = new DisplayPropertiesService(graphQLConfiguration)
 
-      status(response) mustBe FORBIDDEN
+      val controller =
+        new DeleteAdditionalMetadataController(
+          consignmentService,
+          customMetadataService,
+          displayPropertiesService,
+          getValidJudgmentUserKeycloakConfiguration,
+          getAuthorisedSecurityComponents
+        )
+
+      val closureResponse = controller
+        .confirmDeleteAdditionalMetadata(consignmentId, closureMetadataType, fileIds)
+        .apply(FakeRequest(GET, s"/consignment/$consignmentId/additional-metadata/confirm-delete-metadata/$closureMetadataType"))
+
+      val descriptiveResponse = controller
+        .confirmDeleteAdditionalMetadata(consignmentId, descriptiveMetadataType, fileIds)
+        .apply(FakeRequest(GET, s"/consignment/$consignmentId/additional-metadata/confirm-delete-metadata/$descriptiveMetadataType"))
+
```

Comment:
If we use the `displayProperties` to delete both metadata types (I'm referring to my previous comment) then we don't need to call the endpoint and verify the response for both metadata types.
This applies to all the following tests below.

Reply:
Still need to test for each type of metadata because need to ensure the correct properties are deleted depending on the type

Reply:
Ok, thanks